src/odm_sharing/tools/share.py

```diff
@@ -27,6 +27,8 @@ class OutFmt(str, Enum):
     EXCEL = 'excel'
 
 
+# {{{ cli param desc
```

Comment:
Why are you adding the `{{{`?

Reply:
it's foldable vim section syntax. is it ok?